frontend/packages/data-portal/app/components/Run/steps.tsx

```diff
@@ -0,0 +1,235 @@
+          <KeyActionList actions={essentialControls} />
+        </div>
+      </StepContent>
+    ),
+  },
+  {
+    target: getIframeElement('.neuroglancer-layer-group-viewer:has(.neuroglancer-rendered-data-panel)'),
+    title: 'Keyboard shortcuts',
+    placement: 'left',
+    content: (
+      <StepContent variant='simple'>
+        <div className='mt-4 mb-4'>
+          <video autoPlay loop muted playsInline className='rounded'>
+            <source src="/images/keyboard_shortcuts.mp4" type="video/mp4" />
+            <img src="/images/ml-challenge-competition-data-preview.png"
+              title="Your browser does not support the video tag."
+              alt="Movie clips of raw tilt series, tilt series after alignment, and tomogram of a purified lysosome."
+            />
+          </video>
+        </div>
+        <p className='text-[#767676]'>Quickly access top tools with shortcuts. Here are some useful shortcuts:</p>
+        <div className='flex flex-col gap-3 mt-3 mb-4'>
+          <KeyActionList actions={keyboardShortcuts} />
+        </div>
+        <p>These actions are also easily accessible on the top bar.</p>
+      </StepContent>
+    ),
+  },
+  {
+    target: getIframeElement('.neuroglancer-side-panel:has(.neuroglancer-layer-list-panel-items)'),
+    title: 'Layer management',
+    placement: 'right-start',
+    content: (
+      <StepContent variant='simple'>
+        <div className='mt-4 mb-4'>
+          <video autoPlay loop muted playsInline className='rounded'>
+            <source src="/images/layer_management.mp4" type="video/mp4" />
+            <img src="/images/ml-challenge-competition-data-preview.png"
+              title="Your browser does not support the video tag."
+              alt="Movie clips of raw tilt series, tilt series after alignment, and tomogram of a purified lysosome."
+            />
+          </video>
+        </div>
+        <p className='text-[#767676] flex flex-col gap-4'>
+          <span>Data is organized into distinct layers, which all appear in this menu.</span>
+          <span>You can toggle layer visibility, open layer controls or archive (hide from viewer and top layer bar) your layers from here.</span>
+        </p>
+      </StepContent>
+    )
+  },
+  {
+    target: getIframeElement('.neuroglancer-layer-side-panel-tab-view'),
+    title: 'Layer controls',
+    placement: 'right-start',
+    content: (
+      <StepContent variant='minimal'>
+        <p className='text-[#767676]'>
+          Change the visualization settings of individual layers here. 
+          Change <span className='text-black'>layer colours</span> or <span className='text-black'>opacity</span>, 
+          {' '}<span className='text-black'>image contrast limits</span>, 
+          {' '}<span className='text-black'>rendering resolution</span> and more.
+        </p>
+      </StepContent>
+    )
+  },
+  {
+    target: getIframeElement('.neuroglancer-viewer-top-row'),
```

Comment:
For this one, we either need to change the target element, or change the text.

Currently it highlights the neuroglancer top bar. But the text is written as if you are targeting the portal frame header